### DISCUSSION TITLE
easy tutorial please

### DISCUSSION THREAD

**Discussion by Team098** [Subhankar1712/smallchat#1] [General]
Hi, How do I get it to run? need easy tutorial please.

**Comment by Subhankar1712** [comment]
Furthermore, a Makefile is at the root of the repo, usually all build/run instructions are there.

in this specific case running make all or make smallchat would build it, you can then run the produced executable like ./smallchat .
to do that ofc you would need make to be installed.

linux system case:
server:

gcc smallchat.c -o smallchat && ./smallchat


new terminal
client:

nc 127.0.0.1 7711

/nick yourname


This is more than enough, but you really should know about Makefile s because they are pretty common
